easybuild/easyconfigs/p/pypmt/pypmt-1.3.0-gfbf-2024a.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'pypmt'
+version = '1.3.0'
```

please add CUDA versionsuffix if CUDA is a dep:
```suggestion
version = '1.3.0'
versionsuffix = '-CUDA-%(cudaver)s'
```